index.js

```diff
@@ -86,7 +87,6 @@ export const authorize = ({
 
   if (Platform.OS === 'ios') {
     nativeMethodArguments.push(useNonce);
-    nativeMethodArguments.push(usePKCE);
```

Comment:
I know this is not ideal, but could you add the `usePKCE` argument to Android without disrupting the parameter order on iOS? Otherwise this will be a breaking change, and I won't be able to release it until the next major version.

Reply:
I pushed a new commit. I hope I did it correctly. :)

Reply:
Perfect, thanks!